lists/ir.csv

```diff
@@ -1,9 +1,6 @@
 url,category_code,category_description,date_added,source,notes
-http://213.4.130.210,NEWS,News Media,2014-04-15,citizenlab,
 http://4shared.com,FILE,File-sharing,2015-03-20,OONI,
-http://aadaamaak.blogfa.com,LGBT,LGBT,2014-04-15,citizenlab,
 http://aawsat.com/,NEWS,News Media,2016-04-20,citizenlab,
-http://ablogformen2.blogspot.com,LGBT,LGBT,2014-04-15,citizenlab,

```

Comment:
This site also appears to be blocked based on OONI data: https://explorer.ooni.torproject.org/measurement/4F3l2MvLez7ANO9BpmjMHu4c59DRur0l9qr6vCfAMJuhi5wVzNHXIsmUpO1HPNN9?input=http:%2F%2Fablogformen2.blogspot.com.

Note it's not listed in this page: https://explorer.ooni.torproject.org/website/ablogformen2.blogspot.com, because it seems like they changed the blockpage on date 2017-01-08.